osu.Game.Rulesets.Catch/Objects/Drawables/IHasCatchObjectState.cs

```diff
@@ -12,7 +12,7 @@ namespace osu.Game.Rulesets.Catch.Objects.Drawables
     /// </summary>
     public interface IHasCatchObjectState
     {
-        PalpableCatchHitObject HitObject { get; }
+        CatchHitObject HitObject { get; }
```

Comment:
Not sure where this generalisation is coming from - I think this should remain as `Palpable...`? This surely doesn't apply to banana showers or juice streams, right?